client/src/constants/russian.ts

```diff
@@ -0,0 +1,133 @@
+const ruTranslations = {
+  landing_title: 'Добро пожаловать в TaskNote',
+  landing_subtitle: 'Ваш лучший помощник для заметок и задач!',
+  landing_btn_login: 'Войти',
+  landing_btn_register: 'Зарегистрироваться',
+  landing_btn_english: 'English',
+  landing_btn_portuguese: 'Português do Brasil',
+  landing_btn_spanish: 'Español',
+  landing_btn_russian: 'Русский',
+
+  login_title: 'Войти',
+  login_email_label: 'Электронная почта',
+  login_email_placeholder: 'Введите вашу электронную почту',
+  login_password_label: 'Пароль',
+  login_password_placeholder: 'Введите ваш пароль',
+  login_btn_submit: 'Войти',
+  login_account: 'Ещё нет аккаунта?',
+  login_go_other: 'Создать аккаунт',
+  login_back_home: 'Вернуться на главную',
+
+  register_title: 'Создать аккаунт',
+  register_email_label: 'Электронная почта',
+  register_email_placeholder: 'Введите вашу электронную почту',
+  register_password_label: 'Пароль',
+  register_password_placeholder: 'Введите ваш пароль',
+  register_btn_submit: 'Создать аккаунт',
+  register_account: 'Уже есть аккаунт?',
+  register_go_other: 'Войти',
+  register_back_home: 'Вернуться на главную',
+
+  home_nav_home: 'Главная',
+  home_nav_tasks: 'Задачи',
+  home_nav_notes: 'Заметки',
+  home_nav_about: 'О приложении',
+  home_welcome_title: 'Добро пожаловать в вашу панель управления',
+  home_card_task_title: 'Обзор задач',
+  home_card_task_pending: 'Незавершённые задачи',
+  home_card_task_empty: 'Нет незавершённых задач',
+  home_card_task_done: 'выполненные задачи!',
```

Comment:
Why is it not capitalized here? Is it used with a number?

Reply:
Yes, nailed it.

Reply:
Done here https://github.com/ricardo-campos-org/react-typescript-todolist/pull/189/commits/12b10e04de236beb52c8d9e180fc721ef5c1ff6e